packages/flutter_test/lib/src/widget_tester.dart

```diff
@@ -545,6 +548,28 @@ class WidgetTester extends WidgetController implements HitTestDispatcher, Ticker
     }).then<int>((_) => count);
   }
 
+  /// Repeatedly triggers `frameCount` frames with a fixed `frameDuration`.
+  /// 
+  /// The `frameDuration` defaults to 16.683 milliseconds (59.94 FPS).
+  /// All frames stop executing after finishing [EnginePhase.build], except for
+  /// the last frame, which executes the full pass.
```

Comment:
Why the weird behaviour with the EnginePhase? Seems like it's asking for trouble (e.g. what if you have a LayoutBuilder?)

Reply:
I'm concerned about the executing time, which can be dropped from 1.3s to 0.7s in this way (while all other tests, including normal golden tests). Should I remove it?

Reply:
I've removed this behavior. Now all frames are pumped through.